Validator.py

```diff
@@ -0,0 +1,95 @@
+import glob
+import seaborn as sns
+from prompts import *
+
+class Validator:
+    def __init__(self, bad=None, confusion_matrix=None):
+        if bad is None:
```

Comment:
Mnogo jednostavnija implementacija istog konstruktora, obratiti paznju

def __init__(self, bad=0, confusion_matrix=None):
    self.bad = bad
    self.confusion_matrix = confusion_matrix or [[0, 0, 0, 0, 0] for _ in range(5)]